application/GW2 Addon Manager/UI/OpeningPage/OpeningViewModel.cs

```diff
@@ -186,6 +186,7 @@ public string GamePath
                 if (value == _gamePath) return;
                 SetProperty(ref _gamePath, value);
                 _configurationManager.UserConfig.GamePath = value;
+                new Configuration(_configurationManager).DetermineSystemType();
```

Comment:
Instead of creating new configuration object, you could use existing private one, like this: 
```suggestion
               _configuration.DetermineSystemType();
```